evap/staff/tests/test_views.py

```diff
@@ -2037,23 +2133,32 @@ def test_textanswers_showing_up(self):
         self.app.get(self.url, user=self.manager, status=403)
 
         # add additional voter
+        helper_exit_staff_mode(self)
         let_user_vote_for_evaluation(self.app, self.student2, self.evaluation)
 
         # now it should work
+        helper_enter_staff_mode(self)
         self.app.get(self.url, user=self.manager, status=200)
 
     def test_textanswers_quick_view(self):
+        helper_exit_staff_mode(self)
         let_user_vote_for_evaluation(self.app, self.student2, self.evaluation)
+        helper_enter_staff_mode(self)
         page = self.app.get(self.url, user=self.manager, status=200)
         self.assertContains(page, self.answer)
 
     def test_textanswers_full_view(self):
+        helper_exit_staff_mode(self)
         let_user_vote_for_evaluation(self.app, self.student2, self.evaluation)
+        helper_enter_staff_mode(self)
```

Comment:
a python context manager might be a good fit for this pattern. would make it a little easier to not get it wrong in the future!

Reply:
Yes, I use a context manager now. This reads much better :)